doc/radosgw/s3/bucketops.rst

```diff
@@ -91,17 +91,19 @@ Syntax
 Parameters
 ~~~~~~~~~~
 
-+-----------------+-----------+-----------------------------------------------------------------------+
-| Name            | Type      | Description                                                           |
-+=================+===========+=======================================================================+
-| ``prefix``      | String    | Only returns objects that contain the specified prefix.               |
-+-----------------+-----------+-----------------------------------------------------------------------+
-| ``delimiter``   | String    | The delimiter between the prefix and the rest of the object name.     |
-+-----------------+-----------+-----------------------------------------------------------------------+
-| ``marker``      | String    | A beginning index for the list of objects returned.                   |
-+-----------------+-----------+-----------------------------------------------------------------------+
-| ``max-keys``    | Integer   | The maximum number of keys to return. Default is 1000.                |
-+-----------------+-----------+-----------------------------------------------------------------------+
++-----------------+-----------+-----------------------------------------------------------------------------------------------------+
+| Name                | Type      | Description                                                                                     |
++=================+===========+=====================================================================================================+
+| ``prefix``          | String    | Only returns objects that contain the specified prefix.                                         |
++-----------------+-----------+-----------------------------------------------------------------------------------------------------+
+| ``delimiter``       | String    | The delimiter between the prefix and the rest of the object name.                               |
++-----------------+-----------+-----------------------------------------------------------------------------------------------------+
+| ``marker``          | String    | A beginning index for the list of objects returned.                                             |
++-----------------+-----------+-----------------------------------------------------------------------------------------------------+
+| ``max-keys``        | Integer   | The maximum number of keys to return. Default is 1000.                                          |
++-----------------+-----------+-----------------------------------------------------------------------------------------------------+
+| ``allow-unordered`` | Boolean   | Non-standard extension. Allows results to be returned unordered. Cannot be used with delimiter. |
++-----------------+-----------+-----------------------------------------------------------------------------------------------------+
```

Comment:
it doesn't look like this table is rendering correctly: http://docs.ceph.com/ceph-prs/21026/radosgw/s3/bucketops/#id4

i think the `++-----------------+` parts need to be extended to match?

Reply:
Thanks, Casey. They were the right length. But the "+"s weren't aligned correctly.